src/html/fetch-markupconfig.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const { setdefault } = require('ferrum');
+const { MarkupConfig } = require('@adobe/helix-shared');
+
+async function fetchMarkupConfig(context, action) {
+  const { request, downloader } = action;
+
+  const {
+    owner, repo, branch, ref,
+  } = request.params || {};
+  const path = '/helix-markup.yaml';
+
+  // ignore markup config for non-github content
+  if (!owner || !repo) {
+    return;
+  }
+
+  try {
+    // schedule fetch task
+    const res = await downloader.fetchGithub({
```

Comment:
this should not be `await`ed. do this later when you actually need the config.

Reply:
Makes sense!

@trieloff, @tripodsan I'll be busy with various Screens tasks this week, so probably won't get to spend time on this right now. Feel free to take this over if you need it sooner rather than later, otherwise I'll pick this up sometime around Mar 13th